app/src/main/res/layout/auth_url_container.xml

```diff
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android">
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <data>
         <import type="android.view.View" />
+
+        <variable
+            name="action"
```

Comment:
Because it is used by signup and login both layouts. So both will bind different actions on go.